itests/eth_filter_test.go

```diff
@@ -2349,3 +2495,39 @@ func unpackUint64Values(data []byte) []uint64 {
 	}
 	return vals
 }
+
+func deployContract(ctx context.Context, t *testing.T, client *kit.TestFullNode, ethAddr ethtypes.EthAddress,
```

Comment:
what's the difference between this and using `client.EVM().DeployContractFromFilename`?

Reply:
@rvagg The main difference is that `DeployContractFromFilename ` uses Filecoin messages to create the transaction.

I'd like to use ETH messages to create contracts and submit transactions here as we're testing ETH RPC.

Reply:
I think you should document this then, or rename it to make it clear why it's here so a future someone doesn't just come and remove it

Reply:
Done.